src/esp/bindings/GfxBindings.cpp

```diff
@@ -57,10 +57,16 @@ void initGfxBindings(py::module& m) {
   render_camera
       .def(py::init_alias<std::reference_wrapper<scene::SceneNode>,
                           const vec3f&, const vec3f&, const vec3f&>())
-      .def("set_projection_matrix", &RenderCamera::setProjectionMatrix, R"(
-        Set this `Camera`'s projection matrix.
-      )",
-           "width"_a, "height"_a, "znear"_a, "zfar"_a, "hfov"_a)
+      .def("set_projection_matrix",
+           static_cast<RenderCamera& (RenderCamera::*)(int, int, float, float,
+                                                       float)>(
+               &RenderCamera::setProjectionMatrix),
+           R"(Set this `Camera`'s projection matrix.)", "width"_a, "height"_a,
+           "znear"_a, "zfar"_a, "hfov"_a)
```

Comment:
I'm probably annoying with this already, but can the FoV be `Mn::Deg`? :grin: Now that you're redoing most of the API anyway.

Both in Python and C++.

Reply:
Maybe this could wait for the sensor redesign?  I'm not really touching those components (the RenderCamera::setProjectionMatrix I used just sets the matrix).

Reply:
This PR is part of sensor redesign project. :)
Would you do me a favor and solve it here? Otherwise I think it will be forgotten.
Thanks!

Reply:
Ok :)

Reply:
Thanks a lot! @jturner65